### ISSUE TITLE
Accessibility: Colour Palette and Contrast

### ISSUE THREAD

**Issue by FlyteWizard** [VikeLabs/courseup#178]
Hey Folks! 

First off I love the colour palette chosen for the project, however some of the colour palettes don't meet WCAG guidelines. 

This issue should be discussed in depth with the core team, but you can see an overview of the colour palettes that have an issue. 

* Alert Beta Banner: https://contrastchecker.online/ffffff/38a169
* Navigation Header: https://contrastchecker.online/ffffff/82cbee
* Sidebar (Department/Course Selection): https://contrastchecker.online/ffffff/39c686
* Unselected Green Button: https://contrastchecker.online/ffffff/2e8057
* Selected Green Button: https://contrastchecker.online/ffffff/276749
* Hover Green Button: https://contrastchecker.online/ffffff/38a169
* Hover Purple Button: https://contrastchecker.online/ffffff/805ad5
* Red Button: https://contrastchecker.online/ffffff/e53e3e
* Calendar Table Time Light Schedule: https://contrastchecker.online/ffffff/686868

Colours that do not need change and meet criteria: 

* Today/Filter Button: https://contrastchecker.online/1a202c/eff6ff
* Calendar Table Header Days: https://contrastchecker.online/1a202c/eff6ff
* Calendar Table Time Dark Schedule: https://contrastchecker.online/ffffff/4e4e4e
* Unselected Purple Button: https://contrastchecker.online/ffffff/6145ab
* Selected Purple Button: https://contrastchecker.online/ffffff/553c9a

Please feel free to add to either list. Also feel free to correct any errors. 😄
